setup.py

```diff
@@ -36,7 +36,8 @@
         'serenata_toolbox.chamber_of_deputies',
         'serenata_toolbox.datasets'
     ],
+    scripts=['serenata_toolbox/serenata-toolbox'],
     url=REPO_URL,
     python_requires='>=3.6',
-    version='15.0.5',
+    version='15.0.6',
```

Comment:
This is a minor (not a micro) bump: _values have been added, existing values are unchanged_. 15.1.0 then.